Packs/Core/Playbooks/playbook-Unsigned_and_unpopular_process_performed_an_injection.yml

```diff
@@ -0,0 +1,823 @@
+    isautoswitchedtoquietmode: false
+  "5":
+    id: "5"
+    taskid: 7c09ff5c-2f1e-4c55-85f1-557891e3e8f7
+    type: condition
+    task:
+      id: 7c09ff5c-2f1e-4c55-85f1-557891e3e8f7
+      version: -1
+      name: Check if the causality process is signed
+      description: Check if the causality process image is signed.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "14"
+      Signed:
+      - "20"
+    separatecontext: false
+    conditions:
+    - label: Signed
+      condition:
+      - - operator: isNotEqualString
+          left:
+            value:
+              simple: alert.cgosignature
+            iscontext: true
+          right:
+            value:
+              simple: SIGNATURE_SIGNED
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 980,
+          "y": 1150
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "7":
+    id: "7"
+    taskid: 3e3d733d-1317-44cf-8178-e0015cc3b874
+    type: condition
+    task:
+      id: 3e3d733d-1317-44cf-8178-e0015cc3b874
+      version: -1
+      name: Check for known preceding alerts
+      description: Search for commonly triggered alert names preceding the injection alert.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "17"
+      "yes":
+      - "10"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: startWith
+          left:
+            value:
+              simple: foundIncidents.name
+            iscontext: true
+          right:
+            value:
+              simple: Powershell Activity
+          ignorecase: true
+        - operator: isEqualString
+          left:
+            value:
+              simple: foundIncidents.name
+            iscontext: true
+          right:
+            value:
+              simple: Unsigned process injecting into a Windows system binary with no command line
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 560,
+          "y": 790
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "8":
+    id: "8"
+    taskid: 81db7e8a-cc03-44e9-86a5-70d784b286ee
+    type: regular
+    task:
+      id: 81db7e8a-cc03-44e9-86a5-70d784b286ee
+      version: -1
+      name: Close the Alert as True Positive
+      description: commands.local.cmd.close.inv
+      script: Builtin|||closeInvestigation
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "9"
+    scriptarguments:
+      closeReason:
```

Comment:
Please add closeNotes as we have used in all our playbooks. The closed note should be 'True Positive'.